crates/bevy_reflect/src/type_uuid.rs

```diff
@@ -16,4 +18,12 @@ where
     fn type_uuid(&self) -> Uuid {
         Self::TYPE_UUID
     }
+
+    fn display_type(&self) -> String {
```

Comment:
Can we make this a simple `type_name(&self) -> &'static str` getter? That enables callers to format them as they see fit.

Reply:
changed